content/it/cloud_native_apps.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Cloud Native Apps
+status: Completed
+category: Concetto
+---
+
+## Cos'è
+Le applicazioni Cloud Native sono progettate specificamente per sfruttare le innovazioni nel [Cloud Computing](it/cloud_computing/). Queste applicazioni si integrano facilmente con le rispettive architetture cloud sfruttandone le risorse e le capacità in termini di [scalabilità](it/scalability/). La definizione si riferisce anche ad applicazioni che sfruttano le innovazioni in ambito infrastrutturale, introdotte dal Cloud Computing. Con applicazioni Cloud Native oggi si intendono app che vengono eseguite indifferentemente su data center di un provider cloud o su piattaforme Cloud Native on-premise.
```

Comment:
```suggestion
Le applicazioni Cloud Native sono progettate specificamente per sfruttare le innovazioni nel [Cloud Computing](it/cloud_computing/). Queste applicazioni si integrano facilmente con le rispettive architetture cloud sfruttandone le risorse e le capacità in termini di [scalabilità](it/scalability/). La definizione si riferisce anche ad applicazioni che sfruttano le innovazioni in ambito infrastrutturale, introdotte dal Cloud Computing. Con applicazioni Cloud Native oggi si intendono app che vengono eseguite indifferentemente in data center di un provider cloud o su piattaforme Cloud Native on-premise.
```
con datacenter di solito si intende un luogo fisico in cui sono installati dei server.

Reply:
ecco, non so mai come gestirmi con le preposizioni 😅